doc/manual/src/language/import-from-derivation.md

```diff
@@ -0,0 +1,142 @@
+# Import From Derivation
+
+The value of a Nix expression can depend on the contents of a [store object].
+In this case, when that store object is needed, evaluation will be paused, the store object [realised], and then evaluation resumed.
```

Comment:
I think the first paragraph should have a short description of IFD. Currently one needs to read through 3 paragraphs to get to the first definition of IFD.